doc/Programming-guide/Key-platform-features/Data-collection/Oracle-NoSQL-log-appender/index.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: page
+title: Oracle NoSQL log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Oracle-NoSQL-log-appender
+sort_idx: 80
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Oracle NoSQL log appender is responsible for transferring logs from the Operations service to the Oracle NoSQL key/value storage.
+Logs are stored in the storage using the following key path:
+
+```bash
+${applicationToken}/${logSchemaVersion}/${endpointKeyHash}/${uploadTimestamp}/${counter}
+```
+
+The path variables used are:
+
+|Path Variable|Description|
+|---|---|
+|applicationToken|The token of the application|
+|logSchemaVersion|The version of the log schema|
+|endpointKeyHash|The ID of the endpoint the log data belongs to|
+|uploadTimestamp|The timestamp of log upload to the storage (in milliseconds)|
+|count|The log record ID|
+
+<br/>
+
+Values are stored as serialized generic records using record wrapper Avro schema.
+
+# Creating Oracle NoSQL log appender with Admin UI
+
+The easiest way to create an instance of the Oracle NoSQL log appender for the application is by using Admin UI.
+
+To create a log appender of the Oracle NoSQL key/value storage type, do the following:
+
+1. In the **Log appenders** window, click **Add log appender**.
+2. Enter the log appender name and description, select the minimum and maximum supported schema version, select necessary log metadata fields.
+3. Set the log appender type to _Oracle NoSQL_.
+4. Fill in the Oracle NoSQL log appender configuration form.
+5. Click **Add** button. Log appender is ready and operational at this point.
+
+![Add log appender in Admin UI](attach/add-log-appender-in-admin-ui.png)
+
+# Creating Oracle NoSQL log appender with Admin REST API
+
+It is also possible to create an instance of the Oracle NoSQL log appender for the application by using the
+[REST API]({{root_url}}Programming-guide/Server-REST-APIs #TODO). The following example illustrates how to provision the Oracle NoSQL log appender via the
+Admin REST API.
+
+## Configuration
+
+The Oracle NoSQL log appender configuration must match to
+[this](https://github.com/kaaproject/kaa/blob/master/server/appenders/oracle-nosql-appender/src/main/avro/oracle-nosql-appender-config.avsc) Avro schema.
+
+The following configuration example matches the previous schema.
```

Comment:
The following configuration example matches the ~~previous~~ schema.

Here, the schema is referenced by a link in the previous paragraph and such details are not necessary. Could this be rewritten to something like 'An example configuration is as below'?